### ISSUE TITLE
Missing google_app_id. Firebase Analytics disabled.

### ISSUE THREAD

**Issue by Udaychinthala** [kmcgill88/admob_flutter#110]

**Comment by kmcgill88** [comment]
Can you elaborate please. This plugin isn’t for analytics.

**Comment by Udaychinthala** [comment]
Dont we get ads in debug mode if we provide original ad unit ids

**Comment by kmcgill88** [comment]
Yes you should get test ads. I'm not sure how this relates to analytics?

**Comment by Udaychinthala** [comment]
Im not getting ads when i provide my original admob ids 
but im getting when i provide the test ids

**Comment by exptk3** [comment]
Today I saw admob related error logs that contain the following error message as the title of this topic.
 : Missing google_app_id. Firebase Analytics disabled. (In addition, you will see various error messages related to AdMob.)

A few days ago, when I saw an advertisement, I didn't see this error message, but I saw it for the first time today.
So while looking for the reason, I recalled the recent upgrade of the flutter sdk, and when I roll back to the previous sdk version, this error message disappeared.
 - before upgrade : v1.12.13+hotfix.7 --> no error message
 - after upgrade : v1.12.13+hotfix.8 --> showed error message (Missing google_app_id. Firebase Analytics disabled....etc.)

Try to reproduce this bug as I said. You will see a number of AdMob related error messages.

Please find a way to prevent the admob_flutter package from google analytics related error messages after the flutter sdk upgrade.
Thank you.

**Comment by kaymly** [comment]
Same here: Missing google_app_id
admob_flutter: "^0.3.4" on android.

**Comment by floteew** [comment]
Same here. Worked fine when using the test id and get the same error when using the real admob id

**Comment by toldia** [comment]
I have the same error:  Missing google_app_id.

Please, help us!

**Comment by ferdigokdemir** [comment]
Taking same error : 

Missing google_app_id. Firebase Analytics disabled. See https://goo.gl/NAOOOI

GoogleService failed to initialize, status: 10, Missing google app id value from from string resources with name google_app_id

**Comment by vegidio** [comment]
@kmcgill88 Any update on this?

**Comment by termslang** [comment]
Same problem

**Comment by AhmadNazir7576506** [comment]
Hello flutter family....Hope all of you are doing greate....
you can resolve this error  by doing this .......
1: you need to add "admob_app_id" value into ..android/app/google-services.json file, after the "services" tag.

2: "admob_app_id": "ca-app-pub-your app~id" *(*change your admob app id here)
